### PR TITLE
Fixed file paths

### PR DESCRIPTION
**Pull request by saschagrunert**

### CODE REVIEW

**Comment by obnoxxx** [comment]
Thanks for the patch!

File: extras/kubernetes/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ $ kubectl get nodes
 * Deploy the GlusterFS DaemonSet
 
 ```
-$ kubectl create -f gluster-daemonset.json
+$ kubectl create -f glusterfs-daemonset.json
 ```
```
Comment (obnoxxx) [inline extras/kubernetes/README.md]: correct

File: extras/kubernetes/README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Repeat as needed.
 First you will need to deploy the bootstrap Heketi container:
 
 ```
-$ kubectl create -f deploy-heketi-deployment.json
+$ kubectl create -f heketi-deployment.json
```
Comment (obnoxxx) [inline extras/kubernetes/README.md]: It's correct that the deploy-heketi-deployment.json does not exist any more, but the whole text still talks about the bootstrap Heketi container. Somehow this text got broken in the last couple of updates...

**Comment by obnoxxx** [comment]
@raghavendra-talur the change is not wrong, but that hardly makes the document any better at this stage, see my comment above: the whole context is wrong. Also see my comments on #762. I think we need to overhaul the whole readme. It was rendered inconsistent by the last few updates...

**Comment by lpabon** [comment]
Duplicate of #762

**Pull request closed without merging**